windows.md

```diff
@@ -255,9 +255,8 @@ With those compatibility things out of the way, you're ready to start the system
     ```bash
     echo "export PATH=\$PATH:\"/c/Program Files/PostgreSQL/16/bin\"" >> "$USERPROFILE/.bash_profile"
     echo "export PGDATA=\"/c/Program Files/PostgreSQL/16/data\"" >> "$USERPROFILE/.bash_profile"
-    echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> "$USERPROFILE/.bash_profile"
+    echo -e "\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> "$USERPROFILE/.bash_profile"
```

Comment:
seems like these previous `export` commands should be also moved to this line?

Reply:
I decided to have all the `echo` commands for all systems separated into multiple commands as this would make it clear for the students to understand what they are running and what it is doing


de527354234e8dbb4b340f0fa7ac2574e741b3d0